### PR TITLE
Convert to monorepo (Lerna)

### PR DESCRIPTION
**Pull request by lukeed**
**Benefits**

- Runs all tests at once (via `lerna run`)
- Moves main `taskr` repo to `packages/taskr`
- Cuts down most boilerplate code
- Imports a lot of plugins I've authored (**incomplete list**)

**TODOs**

- [ ] Update `taskr` docs
- [ ] Add new `readme.md` for monorepo root
- [x] Fix/Test versioning
- [x] Add more plugins to "official" list
- [x] Update current packages' dependencies

---

- Updated all plugins to ensure Node 4.x (LTS) compatibility 
- Major rewrite of `@taskr/zip` module
- Added `@taskr/less` module

### CODE REVIEW

**Comment by lukeed** [comment]
Whenever you get the chance @hzlmn, would love to hear your initial thoughts on this!

File: package.json [hunk 1]
```diff
@@ -1,56 +1,20 @@
 {
-  "name": "taskr",
-  "version": "0.9.1",
-  "description": "Generator & Coroutine-based task runner. Fasten your seatbelt.",
-  "license": "MIT",
+  "private": true,
   "repository": "lukeed/taskr",
   "homepage": "https://github.com/lukeed/taskr",
```
Comment (hzlmn) [inline package.json:4]: Maybe point directly to repo like `https://github.com/lukeed/taskr/tree/master/packages/taskr` ?

File: packages/clear/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "taskr",
+    "taskr-plugin",
+    "clear",
+    "del",
+    "delete",
+    "rimraf",
+    "remove"
+  ],
+  "dependencies": {
+    "arrify": "^1.0.1",
```
Comment (hzlmn) [inline packages/clear/package.json]: Maybe drop `arrify` it's 2 line of code and seems unnecessary, we can just inline it?

Comment (lukeed) [reply packages/clear/package.json]: Sure. Was going to update & optimize all the deps at once.

File: package.json (hunk 1 above)
Comment (lukeed) [reply package.json:4]: Ah, sure. I'll update the `packages/*` though, not the root `package.json` since it's kind of a dummy/placeholder.

**Comment by lukeed** [comment]
Going to take care of the rest of these items (including `repository` links) after merging. Can also add more official modules as time goes on. 🎉

**Pull request merged**